src/cql/upstream-external/GAD7_LogicLibrary.cql

```diff
@@ -0,0 +1,222 @@
+library GAD7LogicLibrary version '1.0.0'
```

Comment:
@achen2401  did you adapt this from elsewhere? If so, can you please add provenance info to a comment?

Reply:
@mcjustin no,  I didn't adapt it anywhere.  There isn't any existing GAD7 CQL library that I could find on line.